<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ledGadgetGroupBox</class>
 <widget class="QGroupBox" name="ledGadgetGroupBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GroupBox</string>
  </property>
  <property name="title">
   <string>GroupBox</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,2">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="minimumSize">
      <size>
       <width>128</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>140</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>134</width>
        <height>384</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QToolButton" name="btnShowConnect">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/icons/icons/connect-icon.png</normaloff>:/icons/icons/connect-icon.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>128</width>
           <height>64</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnShowBasicSettings">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/icons/icons/wifi.png</normaloff>:/icons/icons/wifi.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>128</width>
           <height>64</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnShowExtraSettings">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/icons/icons/settings.png</normaloff>:/icons/icons/settings.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>128</width>
           <height>64</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnShowLamp">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/icons/icons/lamp.png</normaloff>:/icons/icons/lamp.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>128</width>
           <height>64</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnShowEffects">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/icons/icons/effects.png</normaloff>:/icons/icons/effects.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>128</width>
           <height>64</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QScrollArea" name="scrollArea_2">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>316</width>
           <height>381</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="connectWidget" name="connectW">
            <property name="title">
             <string>Connect Tools</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="editBasicSettingsGroupBox" name="gbBasicSettings">
            <property name="title">
             <string>Basic Settings</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="LampWidget" name="gbLamp">
            <property name="title">
             <string>Lamp</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="editExtraSettingsGroupBox" name="gbExtraSettings">
            <property name="title">
             <string>Extra Settings</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="animationSelectGroupbox" name="gbEffects">
            <property name="title">
             <string>Effects</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
         <zorder>scrollArea</zorder>
         <zorder>gbExtraSettings</zorder>
         <zorder>gbEffects</zorder>
         <zorder>connectW</zorder>
         <zorder>gbBasicSettings</zorder>
         <zorder>gbLamp</zorder>
         <zorder>verticalSpacer</zorder>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>connectWidget</class>
   <extends>QGroupBox</extends>
   <header location="global">connectwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>editBasicSettingsGroupBox</class>
   <extends>QGroupBox</extends>
   <header location="global">editbasicsettingsgroupbox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>editExtraSettingsGroupBox</class>
   <extends>QGroupBox</extends>
   <header location="global">editextrasettingsgroupbox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>animationSelectGroupbox</class>
   <extends>QGroupBox</extends>
   <header location="global">animationselectgroupbox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LampWidget</class>
   <extends>QGroupBox</extends>
   <header location="global">lampwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
